nipype/pipeline/plugins/tests/test_callback.py

```diff
@@ -1,12 +1,20 @@
 # emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
 # vi: set ft=python sts=4 ts=4 sw=4 et:
-"""Tests for workflow callbacks
-"""
+"""Tests for workflow callbacks."""
+from pathlib import Path
 from time import sleep
+import json
 import pytest
 import nipype.interfaces.utility as niu
 import nipype.pipeline.engine as pe
 
+try:
+    import pandas
+
+    has_pandas = True
+except ImportError:
+    has_pandas = False
```

Comment:
Can you just add `pandas` to the test dependencies?

https://github.com/nipy/nipype/blob/1ccee342f6455af20e3c641fdc36caca32ef0b00/nipype/info.py#L154-L163

An explicit minimum version would be appreciated. Pandas 1.2 (first version with Python 3.9 wheels) or 2.0 ([SPEC 0](https://scientific-python.org/specs/spec-0000/)) would be reasonable choices, or anything in between. Shouldn't go later than 2.0 until SPEC0 does.

Reply:
Sure, good idea!

Reply:
LOL, looking back, I suggested that you do this. I'm sorry to jerk you around. Though at a three year lag, it may not qualify as a "jerk".